src/main.py

```diff
@@ -0,0 +1,390 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Jan  1 14:34:11 2025
+@author: David Tiede
+"""
+
+import sys
+import time
+import re
+import os
+from collections import defaultdict
+from pathlib import Path
+import numpy as np
+from PyQt5 import QtCore, QtWidgets, uic
+from functools import partial
+from GUI.ParameterPlot import ParameterPlot
+from GUI.SpectrometerPlot import SpectrometerPlot
+from drivers.CryoDemo import CryoDemo
+from drivers.SpectrometerDemo_advanced import SpectrometerDemo
+from drivers.SLMDemo import SLMDemo
+from drivers.StresingDemo import StresingDemo
+from drivers.MonochromDemo import MonochromDemo
+from DataHandling.DataHandling import DataHandling
+from measurements.MeasurementClasses import AcquireMeasurement,RunMeasurement,BackgroundMeasurement, \
+    ViewMeasurement, KineticMeasurement
+
+
+class MainInterface(QtWidgets.QMainWindow):
+
+    def __init__(self):
+        super(MainInterface, self).__init__()
+        project_folder = Path(__file__).parent.resolve()
```

Comment:
In this line, I changed the way the project folder is determined from using the current working directory to directly using the python script's path